cli/installer/README.md

```diff
@@ -127,3 +182,30 @@ To download and install the "daily" version of azd (most recent build)
 ```bash
 ./install-azd.sh --version daily
 ```
+
+## Uninstall
```

Comment:
Should we also just mention that people can uninstall via their package manager of choice as well (e.g. `brew uninstall`)? I don't think we need to document all the uninstall methods but a mention would be good